__tests__/test.js

```diff
@@ -17,6 +18,82 @@ describe('filterResponseForImportantEvents', () => {
     });
 });
 
+describe('fetchPageOfDataAndFilter', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
+            CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID: '1',
+            GITHUB_IMPORTANT_EVENTS:
+                'CommitCommentEvent,IssueCommentEvent,IssuesEvent,PullRequestEvent,PullRequestReviewEvent,PullRequestReviewCommentEvent'
+        };
+    });
+    it('should resolve with list of important events', () => {
+        nock('https://api.github.com')
+            .get('/test')
+            .reply(200, [
+                { type: 'CommitCommentEvent' },
+                { type: 'NotImportantEvent' },
+                { type: 'PullRequestReviewEvent' },
+                { type: 'AnotherNotImportantEvent' }
+            ]);
+        expect(fetchPageOfDataAndFilter('https://api.github.com/test')).resolves.toEqual(
+            expect.arrayContaining([
+                { type: 'CommitCommentEvent' },
+                { type: 'PullRequestReviewEvent' },
+            ])
+        );
+    });
+    it('should resolve with each important event of same type', () => {
+        nock('https://api.github.com')
+            .get('/test')
+            .reply(200, [
+                { type: 'PullRequestReviewEvent' },
+                { type: 'PullRequestReviewEvent' },
+                { type: 'PullRequestReviewEvent' },
+                { type: 'PullRequestReviewEvent' }
+            ]);
+        expect(fetchPageOfDataAndFilter('https://api.github.com/test')).resolves.toHaveLength(4);
+    });
+    it('should resolve with empty list when no important events found', () => {
+        nock('https://api.github.com')
+            .get('/test')
+            .reply(200, [
+                { type: 'NotImportantEvent' },
+                { type: 'AnotherNotImportantEvent' },
+                { type: 'YetAnotherNotImportantEvent' }
+            ]);
+        expect(fetchPageOfDataAndFilter('https://api.github.com/test')).resolves.toHaveLength(0);
+    });
+});
+
+describe('createIdObject', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
+            CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID: '1',
+            GITHUB_IMPORTANT_EVENTS:
+                'CommitCommentEvent,IssueCommentEvent,IssuesEvent,PullRequestEvent,PullRequestReviewEvent,PullRequestReviewCommentEvent'
+        };
+    });
+    it('should return a properly formatted id object', () => {
+        const row = [123, 456, 789];
```

Comment:
It would help me if this fake data resembled real-world data. So, for example, ['robin-github', 'robin-internal', 'other data'].